test/built-ins/Temporal/now/plainDateTime/calendar-from-undefined.js

```diff
@@ -1,39 +0,0 @@
-// Copyright (C) 2020 Igalia, S.L. All rights reserved.
```

Comment:
Note to self: this was [initially](https://github.com/tc39/proposal-temporal/commit/b5700a83a67ea57c999afaf360342bbb4b2b0336) testing the case where `Calendar.from` was undefined:
```js

Object.defineProperty(Temporal.Calendar, "from", {
  get() {
    actual.push("get Temporal.Calendar.from");
    return undefined;
  },
});
```
Need to check what happened here.